BTW/BurrowsWheelerTransform/BurrowsWheelerTransform/Solution.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using BurrowsWheelerTransform;
+using System.IO;
+
+public class Solution
+{
+    static void Main(string[] args)
+    {
+        string pathToFile = args[0];
+        string text = File.ReadAllText(pathToFile);
+        var (str, index) = StringTransformation.DirectBurrowsWheelerTransformation(text);
+        string answer = StringTransformation.InverseBurrowsWheelerTransformation(str, index);
+        string fileName = Path.GetFileNameWithoutExtension(pathToFile);
+        fileName = pathToFile + "..\\..\\" + fileName + "bwt";
```

Comment:
```suggestion
        fileName = $"{pathToFile}..\\..\\{fileName}bwt";
```
Конкатенировать строки во-первых больше кода, во-вторых, может быть хуже в плане скорости работы, чем использовать интерполяцию строк.